docs/source/terms_and_model.rst

```diff
@@ -1381,23 +1259,20 @@ such as this statement of increased copy number of BRCA1:
   regulatory regions, transcribed regions and/or other functional
   sequence regions.
 
-.. _Quantity:
-
-Quantity
-@@@@@@@@
+Numerics, Comparators, and Ranges
```

Comment:
I prefer that we use the heading **General Purpose Types**. I think we will potentially add to these General Purpose Types over time (or at least until we get through the bulk of the Variation types we need to complete to have a comprehensive spec).  I think "Interval" would also be moved under this heading (and maybe even SequenceExpression.  I think the "identifiable objects" get their own headings for sure as these are more like "Resources" in HL7 FHIR parlance. Why not follow what other standards groups do so as to leverage from what others have learned and gain the benefit of following a plan that we can reference and share.

Reply:
WFM

Reply:
Renamed to Basic Types.